<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventJobTicketsEditor</class>
 <widget class="QDialog" name="EventJobTicketsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="1" column="1">
    <widget class="CApplyResetDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnCloseJobTickets">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Закрыть работы</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSetJobTicket">
      <attribute name="title">
       <string>Назначить</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="CEventJobTicketsView" name="tblSetEventJobTickets"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabChangeJobTicket">
      <attribute name="title">
       <string>Изменить</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="CEventJobTicketsView" name="tblChangeEventJobTickets"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CApplyResetDialogButtonBox</class>
   <extends>QDialogButtonBox</extends>
   <header>library/DialogButtonBox.h</header>
  </customwidget>
  <customwidget>
   <class>CEventJobTicketsView</class>
   <extends>QTreeView</extends>
   <header>Events/EventJobTicketsEditorTable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EventJobTicketsEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EventJobTicketsEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
